package.json

```
@@ -3,11 +3,12 @@
     "version": "1.0.4",
     "private": true,
     "scripts": {
-        "prestart": "yarn run buildplugin && yarn link docusaurus-plugin-navdata",
-        "buildplugin": "cd ./src/plugins/docusaurus-plugin-navdata && yarn install && yarn link",
+        "prestart": "yarn run buildnavplugin && yarn link docusaurus-plugin-navdata && yarn run buildcookiesplugin && yarn link docusaurus-plugin-cookiesbanner",
+        "buildnavplugin": "cd ./src/plugins/docusaurus-plugin-navdata  && yarn install && yarn link",
+        "buildcookiesplugin": "cd ./src/plugins/docusaurus-plugin-cookiesbanner  && yarn install && yarn link",
         "start": "docusaurus start --no-open",
-        "build": "yarn run buildplugin && yarn link docusaurus-plugin-navdata && docusaurus build",
-        "preinstall": "yarn run buildplugin && yarn link docusaurus-plugin-navdata",
+        "build": "yarn run buildnavplugin && yarn link docusaurus-plugin-navdata && yarn run buildcookiesplugin && yarn link docusaurus-plugin-cookiesbanner",
```

Docusaurus' `build` command should be preserved:

```suggestion
        "build": "yarn run buildnavplugin && yarn link docusaurus-plugin-navdata && yarn run buildcookiesplugin && yarn link docusaurus-plugin-cookiesbanner && docusaurus build",
```